install/local/dev/php-fpm/etc/php/99-xdiff.ini

```diff
@@ -1 +0,0 @@
-extension=xdiff
```

Comment:
Since xdiff is only used by the legacy `ODiff` class and the current docker build doesn't actually seem to register the extension, I'm honestly not super interested in keeping it around. We know we're going to replace it with something else.